web/src/Routes.js

```diff
@@ -8,8 +8,12 @@
 // 'src/pages/Admin/BooksPage/BooksPage.js' -> AdminBooksPage
 
 import { Router, Route } from '@redwoodjs/router'
+import supertokens from "supertokens-auth-react";
 
 const Routes = () => {
+  if (supertokens.canHandleRoute()) {
+    return supertokens.getRoutingComponent();
+  }
```

Comment:
This will determine if the current website route can be handled by supertokens (by default, it is `/auth/*`). If yes, we render our login / sign up components, else we return the router block